tetris.py

```diff
@@ -385,37 +365,37 @@ def full_line_clear(self):
         """
         Flashes the line once it's fully populated with blocks and clears it afterwards
         """
-        full_lines = []
+        self.full_lines = []
```

Comment:
Don't put it to `self`, it's weird and doesn't fix any problem

Reply:
Removed, it was an attempt to fix the score